engineering/conventions-go.md

```diff
@@ -24,6 +24,10 @@ This guide documents development conventions for Go at source{d}. Check [general
 * If your project requires fetching or building dependencies, implement that in the `dependencies` rule.
 * If binaries can be built, do so in the `packages` rule. Note that [src-d/ci](https://github.com/src-d/ci) provides that rule by default.
 
+## Docker
+
+* In order to use alpine-based images, your binaries need to be built with `CGO_ENABLED=0`. [src-d/ci](https://github.com/src-d/ci/) does this by default. If your project uses `cgo`, you will have to use alpine for the build of binaries for Docker.
```

Comment:
If I'm not mistaken, to be able to use HTTPS from alpine (and maybe slim as well) you also need to install root certs.